### PR TITLE
MSA-1255: Gidjit - Smart Launcher with SmartThings support

### PR DESCRIPTION
**Pull request by SmartThings**
Submitting 'Gidjit - Smart Launcher with SmartThings support' for publication

### CODE REVIEW

File: smartapps/com-gidjit-smartthings-hub/gidjit-hub.src/gidjit-hub.groovy [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+    oauth: [displayName: "Gidjit", displayLink: "www.gidjit.com"])
+
+
+
+preferences {
+  section ("Allow Gidjit to have access, there by allowing you to quickly control and monitor the following devices") {
+    input "switches", "capability.switch", title: "Control/Monitor your switches", multiple: true, required: false
+    input "thermostats", "capability.thermostat", title: "Control/Monitor your thermostats", multiple: true, required: false
+    input "windowShades", "capability.windowShade", title: "Control/Monitor your window shades", multiple: true, required: false //windowShade
+    //input "bulbs", "capability.colorControl", title: "Control your lights", multiple: true, required: false //windowShade
```
Comment (jasonbio) [inline smartapps/com-gidjit-smartthings-hub/gidjit-hub.src/gidjit-hub.groovy:34]: Remove unused input

**Review by ghost — COMMENTED** [review]
Duplicate of #

**Review by ghost — COMMENTED** [review]
merge:smartapps

**Comment by ghost** [comment]
merge:master

**Pull request merged**